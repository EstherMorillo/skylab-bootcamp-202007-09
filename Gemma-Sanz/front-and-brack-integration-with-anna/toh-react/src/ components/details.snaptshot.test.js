import renderer from 'react-test-renderer';
import React from 'react';
import Details from './details';
import { BrowserRouter as Router } from 'react-router-dom';

describe('App snapshot', () => {
	const props = {
		match: {
			params: {
				heroId: 14
			}
		}
	};
	const tree = renderer.create(
		<Router>
			<Details {...props} />
		</Router>
	);

	it('should match', () => {
		expect(tree.toJSON()).toMatchSnapshot();
	});
});
