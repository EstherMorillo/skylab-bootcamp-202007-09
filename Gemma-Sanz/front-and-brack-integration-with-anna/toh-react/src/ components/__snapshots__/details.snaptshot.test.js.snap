// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot should match 1`] = `
<div
  id="hero-detail__container"
>
  <h2>
    <span
      id="hero-detail__name"
    />
     Details!
  </h2>
  <form
    onSubmit={[Function]}
  >
    <h2>
      Register a new hero:
    </h2>
    <input
      name="heroName"
      onChange={[Function]}
      placeholder="Hero name"
      value={null}
    />
    <p>
      <button
        type="submit"
      >
        Save Hero
      </button>
    </p>
  </form>
</div>
`;
